app/models/viewmodels/decoder/CHIEFError.scala

```diff
@@ -21,28 +21,30 @@ import play.api.Logger
 
 import scala.io.Source
 
-case class CHIEFError(code: String, description: String)
+/** CHIEF errors mapping based on Status DE-code document.
+  * Details can be found in Exports Notifications Behaviour sheet.
+  *
+  * @param code the code value
+  * @param messageKey description for this error code
+  */
+case class CHIEFError(override val code: String, override val messageKey: String) extends CodeWithMessageKey
 
 object CHIEFError {
 
   private val logger = Logger(this.getClass)
-
-  def apply(code: String): Option[CHIEFError] = chiefErrors.find(_.code == code)
+  private val sourcePath = "code_lists/chief_errors.csv"
```

Comment:
Are you sure about this sourcePath?

Reply:
I've changed that in one of the commits. I moved all files for countries and errors to a separate folder.